---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

### Dynamic Runx binding shift and stage-specific gene regulation in early T cell development      

During my postdoctoral training, I studied transcription factor activities and gene regulatory networks in early thymic T cell development. I focused on how Runx transcription factors specifically launch the T-identity program in early T cell development, the vastly distinct roles of Runx factors in other hematopoietic lineages.  By disrupting Runx1 and Runx3 simultaneously using the CRISPR-Cas9 system, I examined the full functional impact of Runx factors, which was previously under-detected due to the functional redundancy. Notably, Runx factors preferentially regulated developmentally dynamic target genes in T-progenitor cells, despite their stable expression kinetics throughout early T-development. In addition, the genes sensitive to loss of Runx factors were substantially different from one stage to the other stage, suggesting highly stage-specific gene regulation. I showed that this switch in regulatory assignments was extensively coordinated with dynamic Runx binding site shifts, as Runx factors were broadly redistributed across the genome, close to their targets, when pro-T cells transited from uncommitted- to the T-committed stages.
[Shin et al., 2021 PNAS](https://www.pnas.org/doi/10.1073/pnas.2019655118)

### Runx-induced T-identity program sensitive to Runx dosage, functional co-factors, and gene network    

The follow-up project addressed why Runx factors occupied substantially different sites in the genome in different stages of T cell development as well as in other hematopoietic contexts: quantitative competition of different partner factors for limiting amounts of Runx protein, yet mostly not restricted by local chromatin state. Thus, when this competition was resolved by moderately increasing the availability of Runx factors, early T cell development was dramatically accelerated via additional Runx binding to post-commitment-specific sites precociously even in the pre-commitment stage. Also, a modest increase in Runx levels controlled the activities of other transcription factors, which propagated Runx-mediated gene regulation through an indirect gene network. These findings showed how Runx factors specifically drove the earliest stages of T cell development, how Runx factors confer context-specific gene regulation, and why these effects were sensitive to Runx expression levels in developing T-progenitor cells.
[Shin et al., 2023 Nat Immunol](https://www.nature.com/articles/s41590-023-01585-z)

### Multi-modular structure of the gene regulatory network for T cell fate specification

I sought to update T cell gene network models, which represent the intricate structure of transcription factor-target gene relationships that direct T cell fate specification. As previous T cell gene network models did not include many of the recently reported datasets, I compiled published data and performed global data analysis.  This analysis revealed a modular structure of the gene regulatory network, in which coherently co-regulated genes formed several distinct modules, and each module operated as a unit of developmental programming.  In addition, this work showed how transcription factors collaborated or opposed each otherâ€™s function within the same module as well as between different modules to shape T cell developmental gene expression programs.  
[Shin and Rothenberg, 2023 Front Immunol](https://www.frontiersin.org/articles/10.3389/fimmu.2023.1108368/full)


### Effector CD4 T cells with progenitor potential mediate chronic intestinal inflammation

In my graduate work, I examined the features of effector CD4 T cells contributing to pathogenesis in chronic inflammatory disorders. I identified a discrete subset of CD4 T cells with gene expression signatures shared with self-renewing T cells and hematopoietic stem and progenitor cells (HSPCs) during chronic intestinal inflammation.  These progenitor-like effector CD4 T cells could self-renew, escape apoptosis, and continually supply proinflammatory CD4 T cells that sustained and conferred intestinal pathology. This work provided an explanation of how pathogenic CD4 T cells mediated the chronicity of inflammation. 
[Shin et al., 2018 J Exp Med](https://rupress.org/jem/article/215/7/1803/42324/Effector-CD4-T-cells-with-progenitor-potential?searchresult=1)

### The role of mitochondrial OXPHOS in shaping the fate decision between Th17 and regulatory T cells

I also examined the effects of mitochondrial oxidative phosphorylation (OXPHOS) controlling pro-inflammatory properties of different subsets of CD4 T cells. I found that ATP-synthase-dependent OXPHOS orchestrated the fate decision between pro-inflammatory Th17 and anti-inflammatory, regulatory T cells. I showed that OXPHOS supported mTOR complex 1 activation in response to the TCR signaling, which was essential for inducing BATF, a key factor for establishing the Th17-chromatin accessibility profile. Consistently, inhibition of OXPHOS during Th17 differentiation prevented Th17-mediated neuroinflammation in the mouse model of multiple sclerosis. 
[Shin et al., 2020 Cell Rep](https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30031-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124720300310%3Fshowall%3Dtrue)

*Complete list of published work:* [Google Scholar](https://scholar.google.com/citations?user=KRA8RWgAAAAJ&hl=en), [NCBI My Bibliography](https://www.ncbi.nlm.nih.gov/myncbi/boyoung.shin.1/bibliography/public/)


